config/missing-page-redirector.php

```diff
@@ -108,14 +108,14 @@
         /*
          * forums
          */
-        '/forum.php' => '/forums/category/{c}',
+        '/forum.php' => '/forums/category/{c}', // TODO what if no category is specified?
```

Comment:
The way it's implemented, if `c` is not in the query params, it won't get replaced, so it'll try to open `/forums/category/{c}` and fail.

The redirector is not used as long as the legacy blade still exists, so it's not a problem that needs to be addressed now, but the solution is to implement a multi-redirect like I did for ticketmanager.php:
```
        '/forum.php' => [
            'c' => '/forums/category/{c}',
            '' => '/forums',
        ],
```